components/avatar/avatar.html

```diff
@@ -62,7 +62,7 @@ <h3>Below is an example for Image Avatar</h3>
           </div>
           <ul class="list-subtopic">
             <li>
-              <a href="#"> Introduction </a>
+              <a href="/get started/introduction.html"> Introduction </a>
```

Comment:
You should not use spaces in the folder name

Reply:
sure will make the changes thank you